lib/Onnxifi/Base.cpp

```diff
@@ -17,13 +17,52 @@
 
 #include "glow/Importer/ONNXIFIModelLoader.h"
 
+#include "llvm/ADT/ScopeExit.h"
 #include "llvm/ADT/SmallVector.h"
 
 namespace glow {
 namespace onnxifi {
+onnxStatus BackendId::checkGraphCompatibility(const void *onnxModel,
```

Comment:
how do we ensure thread safety here?

Reply:
Ah good point, we'll need some locking

Reply:
No locking needed, just created a fresh Module